expo-maps/ios/ExpoMaps/AppleMaps/AppleMapsGeoJsons.swift

```diff
@@ -0,0 +1,48 @@
+import MapKit
+
+class AppleMapsGeoJsons: GeoJsons {
+
+  private let mapView: MKMapView
+  private var annotations: [MKAnnotation] = []
+  private var overlays: [MKOverlay] = []
+
+  init(mapView: MKMapView) {
+    self.mapView = mapView
+  }
+
+  func setGeoJsons(geoJsonObjects: [GeoJsonObject]) {
+    deleteGeoJsons()
+    if #available(iOS 13.0, *) {
```

Comment:
@lukmccall How should we handle the situation in which Apple Maps feature supports iOS versions > 13. What are the best practices in such case?

Reply:
Hmm, that is a complex topic, but long story short, in that case, I would do the same as you did. Only handle JSON when the iOS version supports it. However, I would also add a js method to indicate if that feature is available. You can do it via RN API - https://reactnative.dev/docs/platform-specific-code#detecting-the-ios-version.

Reply:
Thank you!